agent/connect/ca/plugin/provider.proto

```diff
@@ -77,6 +78,11 @@ message CrossSignCAResponse {
     string crt_pem = 1;
 }
 
+// RPCs can't just return a plain bool it seems...
```

Comment:
I wonder if they can return any scalar types?

Reply:
Hmm it actually is possible to define scalar message types per https://developers.google.com/protocol-buffers/docs/proto#scalar but it's not our convention here presumably as this is more forwards compatible if we ever need more so I'll leave it and just update the comment.

Reply:
Looks like the gRPC generator only will support message types and not primitive types: https://github.com/golang/protobuf/blob/ed6926b37a637426117ccab59282c3839528a700/protoc-gen-go/descriptor/descriptor.pb.go#L1220-L1221